app/controllers/user_controller.rb

```diff
@@ -0,0 +1,44 @@
+post '/users/signup' do
+  @user = User.new(:name => params[:user][:name], :password => params[:user][:password])
```

Comment:
FYI the rocket `=>` is starting to go out of style

Reply:
Modified in new push.